routes/api.php

```diff
@@ -22,6 +22,8 @@
     $router->post('user/resetPassword', ['uses' => 'Auth\ResetPasswordController@reset']);
     $router->post('contact/sendMessage', ['uses' => 'ContactController@sendMessage']);
 
+    $router->apiResource('wiki', 'PublicWikiController');
```

Comment:
Not sure if this needs stricter throttling than the default one? Current settings allows for 45 request to be made in one minute per IP. I am not sure however if we actually see the forwarded IP or if this is effectively a global setting, throttling the "ingress client".